examples/basic-click-counter/Index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>React click conter example</title>
+    <meta http-equiv='Content-type' content='text/html; charset=utf-8'>
+    <script src="../../build/react.js"></script>
+    <script type="text/javascript">
+    var Counter = React.createClass({
+      getInitialState: function () {
+        return { clickCount: 0 };
+      },
+      handleClick: function () {
+        this.setState(function(state) {
+          return {clickCount: state.clickCount + 1};
+        });
+      },
+      render: function () {
+        return React.DOM.h2({onClick: this.handleClick},
```

Comment:
Yea, please use JSX and put all the JS together instead of in 2 script tags.

Also, this should be index.html (lowercase).

Also also, I'd prefer if you squashed commits but not a huge deal if you don't know how.